utils/AppUtils.ts

```diff
@@ -67,11 +68,12 @@ export const pingNode = async (
  * @param show
  */
 export const selectedOffer = async (
-  bolt12: string,
+  offer: OfferInfo,
   show: (visible: boolean, message: string) => void
 ) => {
   try {
-    return bolt12;
+    console.log(offer.bolt12);
+    return offer.bolt12;
   } catch (e) {
     show(true, `Error with message: ${e}`);
     console.error(e);
```

Comment:
this function have no sense, it is different from the ping function, the ping function run an http request